packages/cln_plugin/lib/src/rpc_method/rpc_command.dart

```diff
@@ -1,46 +1,13 @@
 import 'package:cln_plugin/cln_plugin.dart';
 
 class RPCCommand {
-  /// Method name
-  String name;
-
-  /// Usage of this method
-  String usage;
-
-  /// Description of this method
-  String description;
-
-  /// The long description of this method
-  late String longDescription;
-
-  bool deprecated;
-
   /// The callback function
   Future<Map<String, Object>> Function(Plugin, Map<String, Object>) callback;
 
-  RPCCommand(
-      {required this.name,
-      required this.usage,
-      required this.description,
-      required this.callback,
-      this.deprecated = false,
-      String? longDescription}) {
-    // means if is null set the longDescription = description
-    this.longDescription = longDescription ?? description;
-  }
+  RPCCommand({required this.callback});
```

Comment:
I will add in this class also 

  `Map<String, dynamic> toMap();`

So the other subclass can convert the class in the correct map